Providers/Crashlytics/AnalyticsKitCrashlyticsProvider.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+
+class AnalyticsKitCrashlyticsProvider: NSObject, AnalyticsKitProvider {
+
+    // Lifecycle
+    func applicationWillEnterForeground() { }
+    func applicationDidEnterBackground() { }
+    func applicationWillTerminate() { }
+    func uncaughtException(exception: NSException) { }
+
+    // Logging
+    func logScreen(screenName: String) {
+        clsLog("screen: \(screenName)")
+    }
+
+    func logEvent(event: String) {
+        clsLog("event: \(event)")
+        Answers.logCustomEventWithName(event, customAttributes: nil)
+    }
+
+    func logEvent(event: String, withProperty key: String, andValue value: String) {
+        logEvent(event, withProperties: [key: value])
+    }
+
+    func logEvent(event: String, withProperties properties: [String: AnyObject]) {
+        clsLog("event: \(event) properties: \(properties)")
+        Answers.logCustomEventWithName(event, customAttributes: properties)
+    }
+
+    func logEvent(event: String, timed: Bool) {
```

Comment:
Should these call `logEvent` if `timed` is false? It seems like long term there would just be one logEvent API with optional properties and timed params that default to nil and false.